### PR TITLE
Alpacas

### PR DESCRIPTION
**Pull request by Gujsingh**

### CODE REVIEW

File: frontend/src/components/StockPage/services/polygon.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    const api = apisauce.create({
+        baseURL: config.Stock_url,
+        headers: {
+            'APCA-API-KEY-ID': config.alpacas_api_key_id,
+            'APCA-API-SECRET-KEY': config.alpacas_api_secret_key
+        },
+    })
+    const getBars = (symbol,start,end,limit,timeframe,adjustment,feed) => api.get('/v2/symbol/{symbol}/bars/?start={start}&end={end}&limit={limit}&timeframe={timeframe}&adjustment=adjustment&feed=feed')
+    const quote = (symbol) => api.get('/v2/stocks/{symbol}/quotes/latest')
```
Comment (nreusken) [inline frontend/src/components/StockPage/services/polygon.js:14]: I think this should be api.get('/v2/stocks/' +symbol + '/quotes/latest'). Same for the other functions

**Review by nreusken — CHANGES_REQUESTED** [review]

File: frontend/src/components/StockPage/alpacas.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      const alpacaApi = alpacaApi()
+      alpacaApi.quote("apple").then((response()=> {
+        console.log("apple stock is worth")
+        console.log(response)
+
+      }
+
+    }
+    render() {
+        retrun <View>
```
Comment (ojas1542) [inline frontend/src/components/StockPage/alpacas.js:18]: return?

**Review by ojas1542 — CHANGES_REQUESTED** [review]
I'm seeing syntax errors in alpacas.js, login.js and register.js

File: frontend/src/components/login/register.js [hunk 3]
```diff
@@ -2,43 +2,69 @@ import React from "react";
...
+                    fetch('/auth/signup', {
+                      method: 'POST',
+                      body: JSON.stringify({
+                        f_name = fName,
+                        l_name =lName,
+                        email = email,
+                        username = username,
+                        p_word = pass,
+                        verify_p_word = verify_password,
+                      })
```
Comment (ojas1542) [inline frontend/src/components/login/register.js:58]: This won't compile. Should be in the form f_name:fName

**Review by ojas1542 — CHANGES_REQUESTED** [review]
After fixing the body in register.js I still get the following console errors when trying to run the login and register page.

Screen Shot 2022-
<img width=
02-06 at 1 43 23 PM" src="https://user-images.githubusercontent.com/17942486/152702614-d4a88d8c-3084-4277-9d57-a45200acbcf6.png">

File: frontend/src/components/login/register.js [hunk 4]
```diff
@@ -13,32 +20,48 @@ export class Register extends React.Component {
...
+                        "first_name": this.state.fName,
+                        "last_name":this.state.lname,
+                        "email": this.state.email,
+                        "username": this.state.username,
+                        "password": this.state.pass,
+                        "verify_password": this.state.verify_password
+                      })
+                    } )
+                   }
+                   console.log(this.state)
```
Comment (ojas1542) [inline frontend/src/components/login/register.js:63]: The server response also needs to be checked to see if account creation was successful cuz I'm not seeing any error messages when I incorrectly fill out the fields.

File: frontend/src/components/login/login.js [hunk 5]
```diff
@@ -13,16 +18,32 @@ export class Login extends React.Component {
...
+                        body: JSON.stringify({
+                          "username":this.state.username,"password":this.state.pass
+                        })
+                      } ).then(
+                        res => res.json()
+                        ).then(
+                          token => {
+                            console.log(this.state)
+                            this.setState({token: token});
+                            console.log(token)
```
Comment (ojas1542) [inline frontend/src/components/login/login.js:43]: the console.log statements are good for debugging but I feel like they should be commented out when pushing to the main branch

**Review by ojas1542 — CHANGES_REQUESTED** [review]

**Review by ojas1542 — APPROVED** [review]

**Review by ojas1542 — APPROVED** [review]
Looking good. All the console.log statements are good for debugging but should be ideally commented out when merging. The server response also needs to be checked when creating an account but these changes aren't super important so we can just have them be part of a future pr.

**Review by nreusken — APPROVED** [review]
Looks good for merge. I agree with the comments made by Ojas!

**Pull request merged**